payments/vipps/redirect.md

```diff
@@ -16,40 +16,31 @@ sidebar:
       title: Other Features
 ---
 
+
 {% include alert-development-section.md %}
 
->Vipps is a two-phase payment instrument supported by the major norwegian banks.
- In the redirect to Swedbank Pay Payments scenario,  Swedbank Pay
- receives a mobile number (MSISDN) from the payer through Payex Payments.
- Swedbank Pay performs a payment that the payer must confirm through the
- Vipps mobile app.
+{% include jumbotron.html body="**Vipps Payments** is a two-phase
+payment instrument supported by the major Norwegian banks. In the redirect
+scenario, Swedbank Pay receives a mobile number (msisdn)
+from the payer through Swedbank Pay Payments. Swedbank Pay performs a payment
+that the payer must confirm through the Vipps mobile app." %}
 
 ## Introduction
 
 * When the payer starts the purchase process, you make a `POST` request towards
   Swedbank Pay with the collected `Purchase` information.
-  This will generate a payment object with a unique `paymentID`.
-  You either receive a Redirect URL to a hosted page or a JavaScript source
+* This will generate a payment object with a unique `paymentID`.
+* You will either receive a Redirect URL to a hosted page or a JavaScript source
   in response.
 * You need to [redirect][reference-redirect] the payer to the Redirect payment
-  page or embed the script source on you site to create a
-  [Hosted View][hosted-view] in an `iFrame`; where she is prompted to enter the
-  registered mobile number.
-  This triggers a `POST` towards PayEx.
+  where the user is prompted to enter the registered mobile number.
+  This triggers a `POST` towards Swedbank Pay.
 * Swedbank Pay handles the dialogue with Vipps and the consumer confirms the
   purchase in the Vipps app.
-* If CallbackURL is set you will receive a payment callback when the Vipps
+* If `callbackURL` is set, the user will receive a payment callback when the Vipps
```

Comment:
This is incorrect. The `callbackUrl` is provided by the merchant to us, and we perform an HTTP `POST` against that URL for every transaction event related to the payment. The user may perform actions that will lead to the `callbackUrl` being requested, but the user will never see or receive the callback itself.

Reply:
I took callbackurl out of introduction because its under the General tab further down on the page. also changed optional to strongly recommended. Let me know if you want it in the intro, but i think its better where it is now ? :)